Sming/Libraries/ArduinoJson6/component.mk

```diff
@@ -1 +1,2 @@
 COMPONENT_SUBMODULES := ArduinoJson
+COMPONENT_INCDIRS += ArduinoJson/src
```

Comment:
Shouldn't this be `:=` instead of `+=` ?

Reply:
It's equivalent to `COMPONENT_INCDIRS := include ArduinoJson/src`. I can change it if you prefer?

Reply:
Yes, please.